gatsby-node.js

```diff
@@ -236,15 +240,15 @@ exports.onCreateNode = async ({ node, boundActionCreators }) => {
 
   const markdownToHtml = util.promisify(remark().use(html).process)
 
-  const hydrateImages = async (node, nameProvider, addDummyImages = false) => {
+  const hydrateImages = async (node, addDummyImages = false) => {
     if (node.images) {
+      let hydratedImages = []
       node.hydratedImages = await (
-        Promise.all(
-          node.images
-            .filter(image => image.file)
-            .map((image, idx) => hydrateImage(image, idx, nameProvider))
-        )
-        .filter(item => !!item)
+        node.images
+          .filter(image => image.file)
+          .reduce((acc, image) => acc.then(() => hydrateImage(image).then((hydratedImage) => hydratedImages.push(hydratedImage))), Promise.resolve())
```

Comment:
nice

Reply:
This is to prevent Cloudinary from complaining that we are hitting it up too fast - even doing it serially we can get through all the images in ~12s